grim_test.go

```diff
@@ -15,6 +19,71 @@ import (
 var testOwner = "MediaMath"
 var testRepo = "grim"
 
+var testHookEvent = hookEvent{
+	Owner: testOwner,
+	Repo: testRepo,
+	EventName: "push",
+}
+var testEffectiveConfig = &effectiveConfig{
+	pendingTemplate: "pending {{.Owner}}",
+	errorTemplate:   "error {{.Repo}}",
+	failureTemplate: "failure {{.Target}}",
+	successTemplate: "success {{.UserName}}",
+	hipChatToken: "NOT_EMPTY",
+	hipChatRoom: "NON_EMPTY",
+}
+
+func TestOnHipChatLoggingGrimError(t *testing.T) {
+	var buf bytes.Buffer
+	logger := log.New(&buf, "", log.Lshortfile)
+
+	//onHook(string,*effectiveConfig, hookEvent, action hookAction, logger *log.Logg
```

Comment:
Another comment that cuts off.  Also it doesn't seem to inform of anything so perhaps it should be deleted.